contrib/win32/openssh/install-sshd.ps1

```diff
@@ -39,13 +39,19 @@ if (Get-Service ssh-agent -ErrorAction SilentlyContinue)
    sc.exe delete ssh-agent 1>$null
 }
 
-# unregister etw provider
-wevtutil um `"$etwman`"
+# Unregister etw provider
+# PowerShell 7.3+ has new/different native command argument parsing
+if ($PSVersiontable.PSVersion -le '7.2.9') {
+    wevtutil um `"$etwman`"
+}
+else {
+    wevtutil um "$etwman"
+}
 
 # adjust provider resource path in instrumentation manifest
 [XML]$xml = Get-Content $etwman
-$xml.instrumentationManifest.instrumentation.events.provider.resourceFileName = $sshagentpath.ToString()
-$xml.instrumentationManifest.instrumentation.events.provider.messageFileName = $sshagentpath.ToString()
+$xml.instrumentationManifest.instrumentation.events.provider.resourceFileName = "$sshagentpath"
```

Comment:
Why are you making this change? the ToString method seems to work fine on 7.3. Is this for readability?

Reply:
First `$sshagentpath` variable is already a string type, but the PowerShell way of to convert to string is using double quotes, which perform string expansion.